src/solvers/flattening/boolbv.cpp

```diff
@@ -302,7 +312,7 @@ bvt boolbvt::convert_bitvector(const exprt &expr)
   else if(expr.id()==ID_float_debug1 ||
           expr.id()==ID_float_debug2)
   {
-    assert(expr.operands().size()==2);
+    DATA_INVARIANT(expr.operands().size() == 2, "");
```

Comment:
There are no expression types corresponding to `ID_float_debug1` and `ID_float_debug2` as it seems they have been added quickly for debugging purposes. Should there be actual expression types for these?

Reply:
I think this can be removed.